PyRDF/backend/Local.py

```diff
@@ -51,7 +51,9 @@ def execute(self, generator):
 
         rdf = ROOT.ROOT.RDataFrame(*generator.head_node.args) # Create RDF object
 
-        values, nodes = mapper(rdf) # Execute the mapper function
+        values = mapper(rdf) # Execute the mapper function
+        nodes = generator.get_action_nodes()
+        # Get the action nodes in the same order as values
```

Comment:
Should this comment precede (or appear next to) the previous line?

Reply:
It should appear on line 56 after `nodes = generator.get_action_nodes()`. I put it below as that line would get too long in that case

Reply:
Then I would put the comment in the line above.

Reply:
I'll make the change @etejedor  👍